### PR TITLE
Add error response

### PR DESCRIPTION
**Pull request by kayladnls**
This PR: 
- [x] adds the ability to create error messages with error codes other than 404, as well as with custom bodies.

### CODE REVIEW

File: src/Piston.php [hunk 1]
```diff
@@ -105,9 +105,14 @@ public function launch()
 
     public function notFound()
     {
-        $this->response->getBody()->write('{}');
+        return $this->getErrorResponse(404);
```
Comment (localheinz) [inline src/Piston.php]: @kayladnls 

Wondering, should we use [`shrikeh/teapot`](https://github.com/shrikeh/teapot) here, too?

Comment (kayladnls) [inline src/Piston.php]: @rcatlin this is the only place where a code is used. Is it worth pulling in the dep for that? I could define a constant.... or?

**Comment by kayladnls** [comment]
@localheinz ping for final code review?

File: src/Piston.php (hunk 1 above)
Comment (rcatlin) [inline src/Piston.php]: Yea, let's pull in the dependency in this PR.

File: src/Piston.php [hunk 2]
```diff
@@ -105,9 +105,14 @@ public function launch()
...
+    }
+
+    public function getErrorResponse($code, $body = '{}')
```
Comment (localheinz) [inline src/Piston.php]: @rcatlin 

Can we add a docblock here? That is, unless this method implements an interface (where we have a docblock).

**Comment by localheinz** [comment]
@kayladnls 

Except for the missing docblock, this looks good to me!

Feel free to merge when you added it and the build passes!

**Pull request merged**